src/ucp/proto/proto_am.inl

```diff
@@ -20,6 +20,24 @@
 
 #define UCP_STATUS_PENDING_SWITCH (UCS_ERR_LAST - 1)
 
+#define UCP_AM_BCOPY_HANDLE_STATUS(_multi, _status) \
+   ({ \
```

Comment:
why `(` ?

Reply:
yes, not needed here
`do {} while (0)` is better here